### DISCUSSION TITLE
关于时区问题

### DISCUSSION THREAD

**Discussion by MengNianxiaoyao** [YunYouJun/valaxy#341] [Q&A]
设置了博客的时区为`Asia/Shanghai`后，发现`VCLiveTime`和`time_warning`的时区是`GMT+0:00`，不是`GMT+8:00`，一定要精确到时分秒时区才会是`GMT+8:00`

**Comment by YunYouJun** [comment]
Fixed in https://github.com/YunYouJun/valaxy/commit/5668599091bd311baf06f1168cd3ef9815441264
